packages/loader/index.js

```diff
@@ -1,27 +1,39 @@
 const {getOptions} = require('loader-utils')
-const mdx = require('@mdx-js/mdx')
+const {createCompiler} = require('@mdx-js/mdx')
 
 const DEFAULT_RENDERER = `
 import React from 'react'
 import {mdx} from '@mdx-js/react'
 `
 
+const pragma = `
+/* @jsxRuntime classic */
+/* @jsx mdx */
+/* @jsxFrag mdx.Fragment */
+`
```

Comment:
why does pragma need to move to this level? :thinking:

Reply:
As can be seen [here](https://github.com/mdx-js/mdx/blob/main/packages/mdx/index.js#L46), the pragma is added by the `compile` function, not by the processor. Because the processor is now invoked by the loader directly instead of the `compile()` function, the pragma needs to be added by the loader manually.